src/main/java/minegame159/meteorclient/modules/combat/AutoAnvil.java

```diff
@@ -143,6 +142,7 @@ public void onActivate() {
                 if (getFloorSlot() == -1) return;
                 mc.player.inventory.selectedSlot = getFloorSlot();
 
+                if (rotate.get()) RotationUtils.packetRotate(target.getBlockPos());
```

Comment:
i didn't add this because i thought sending 2 rotation packets like this would not be beneficial, i dont think there will be issues with not rotating to this placement